CenterForOpenScience/ember-osf-web
# [ENG-2308] Overview route within moderation

- Ticket: [ENG-2308]
- Feature flag: n/a

## Purpose
- Create a view of the overview page for moderators to accept/reject a registration

## Summary of Changes
- Add a query param to the `registries.overview` route called `viewMode`
  - when `?viewmode=moderator`, `overview-header` checks if the current user is a moderator on the provider, and adjusts the `overview-topbar` to show the moderator controls
- Added tests to `topbar-test.ts` to make sure the moderator dropdown is only shown when appropriate


## Side Effects
`NA`

## QA Notes
- Please check this page for good behavior in differing screen sizes and also to make sure that the proper registration info is showing. Only moderators should be able to see the `make decision` button and the subsequent dropdown.

[ENG-2308]: https://openscience.atlassian.net/browse/ENG-2308
[ENG-2183]: https://openscience.atlassian.net/browse/ENG-2183
